docs/markdown/Reference-manual.md

```diff
@@ -372,6 +372,10 @@ following.
 - `install_mode` *(since 0.47.0)*: the file mode and optionally the
   owner/uid and group/gid
 - `output`: list of output files
+- `env` *(since 0.57.0)*: environment variables to set, such as `['NAME1=value1',
+   'NAME2=value2']`, or an [`environment()`
+   object](#environment-object) which allows more sophisticated
+   environment juggling. A dictionary is also accepted.
```

Comment:
since dicts are always supported, maybe reword this the opposite way?
```markdown
environment variables to set, such as `{'NAME1': 'value1', 'NAME2': 'value2'}`, or ... A list of strings in the form `NAME=value` is also accepted.
```

Reply:
I really feel like in the long run we probably want to deprecate the `['NAME=value']` syntax, as it's pretty magical and dicts or `environment()` objects are much nicer to work with anyway.

Reply:
Right, better have the dict syntax first since that's what we should recommend now, made that change now. We could indeed deprecate the string list format, but I preferred to keep it here for consistency with other APIs.